pkgs/development/compilers/gcc/8/default.nix

```diff
@@ -290,4 +298,10 @@ stdenv.mkDerivation ({
 }
 
 // optionalAttrs (enableMultilib) { dontMoveLib64 = true; }
+
+// optionalAttrs stdenv.targetPlatform.isXtensa {
+  postPatch = ''
+    cp -RT ${espressifXtensaOverlays targetPlatform}/gcc .
+  '';
+}
```

Comment:
This should be inlined.

Reply:
The context above suggests otherwise

Reply:
So we take out everything including shebang patching? I think this is worth a comment that this is really what we want.

Reply:
I now see it. Inlined to `prePatch`.